float32 left_front
float32 left_back
float32 right_front
float32 right_back